hyperspy/utils/peakfinders2D.py

```diff
@@ -27,6 +27,16 @@
 
 NO_PEAKS = np.array([[np.nan, np.nan]])
 
+def _get_intensity(X, f, **kwargs):
```

Comment:
```suggestion
def _get_peak_position_and_intensity(X, f, **kwargs):
```
to improve readability, use a name that describe slightly better what it does.

Can you add a docstring?